pkg/deployments/tasks/20221115-aave-rebalanced-linear-pool/readme.md

```diff
@@ -0,0 +1,16 @@
+# 2022-11-15 - Aave Rebalanced Linear Pool
```

Comment:
The deprecated readme should have a deprecation notice and a link to this task (see other readmes for reference)